scripts/data_files/psa_spec/crypto.h

```diff
@@ -0,0 +1,562 @@
+typedef /* implementation-defined type */ psa_aead_operation_t;
```

Comment:
This is generated from a commit which is slightly anterior to the `psa-crypto-api-1.0.1` tag, and slightly different (1.0.1 has additional lifetime-related macros). Let's base our copy on the official released version.